receiver/kubeletstatsreceiver/README.md

```diff
@@ -23,6 +23,11 @@ TLS tells this receiver to use TLS for auth and requires that the fields
 ServiceAccount tells this receiver to use the default service account token
 to authenticate to the kubelet API.
 
+Note: a missing or empty `endpoint` will cause the hostname on which the collector
+is running to be used as the endpoint. If the hostNetwork flag is set, and the
+collector is running in a pod, this hostname will resolve to the node's network
+namespace.
+
```

Comment:
Can we add a `serviseAccount` usage example here with downward API?

If I understand correctly it should be:

```yaml
  kubeletstats:
    authType: serviceAccount
    endpoint: ${K8S_NODE_NAME}:10250
```

Another note: 

Make sure that it's set using the downward API in the collector pod spec as follows:
```yaml
env:
  - name: K8S_NODE_NAME
    valueFrom:
      fieldRef:
        fieldPath: spec.nodeName
```

Reply:
Yeah, that's a good idea. Done.